Makefile

```diff
@@ -102,9 +102,11 @@ branding:
 	sed -i 's/theforeman-foreman/redhat-satellite-operations/g' .github/workflows/*.yml
 	sed -i 's/Foreman Operations Collection/Red Hat Satellite Operations Collection/g' docs/index.rst docs/conf.py
 	sed -i 's/The Foreman Project/Red Hat, Inc./g' docs/conf.py
+	sed -i 's/Foreman/Satellite/g' roles/*/README.md roles/*/*/*.yml
 	sed -i '/FOREMAN_\w/ s/FOREMAN_/SATELLITE_/g' Makefile
 	sed -i '/foreman_proxy_\w/ s/foreman_proxy_/satellite_capsule_/g' roles/*/README.md roles/*/*/*.yml
-	sed -i '/foreman_\w/ s/foreman_/satellite_/g' roles/*/README.md roles/*/*/*.yml
+	sed -i '/foreman_\w/ s/foreman_/satellite_/g' roles/*/README.md roles/*/*/*.{yml,j2}
+	sed -i 's/satellite_rh_cloud/foreman_rh_cloud/g' roles/*/README.md roles/*/*/*.{yml,j2}
```

Comment:
I must admit I *prefer* explicitly writing `roles/*/*/*.yml roles/*/*/*,j2`, but wouldn't block on this.

Reply:
> I must admit I _prefer_ explicitly writing `roles/*/*/*.yml roles/*/*/*,j2`, but wouldn't block on this.

Do you have a reason why? Cause I almost went that route and then I said, someone will tell me this is too verbose and to use the power of the language.

Reply:
It's a bashism, which doesn't necessarily work with other shells, and you don't have control which shell the Makefile ends up running under:

```console
root@d94e9f16960d:/tmp/l# touch a.yml
root@d94e9f16960d:/tmp/l# touch a.j2 
root@d94e9f16960d:/tmp/l# ls *.{yml,j2}
a.j2  a.yml
root@d94e9f16960d:/tmp/l# /bin/dash
# pwd
/tmp/l
# ls *.{yml,j2}
ls: cannot access '*.{yml,j2}': No such file or directory
```